iac-extensions/spring-config/src/main/antlr/PropertiesLexer.g4

```diff
@@ -8,17 +28,20 @@ public PropertiesLexer(CharStream input, boolean crLexerCostructor) {
 }
 
 COMMENT         : [!#] -> pushMode(COMMENT_MODE);
-LEADING_SPACING : {this.getCharPositionInLine() == 0}? [ \t\f\r\n\u2028\u2029]+ -> channel(HIDDEN);
-NEWLINE         : [\r\n\u2028\u2029]+;
+NEWLINE         : [ \t\f\r\n\u2028\u2029]+ -> channel(HIDDEN);
 DELIMITER       : [ ]* [:=\t\f ] [ ]* -> pushMode(VALUE_MODE);
-SLASH           : '\\' -> more, pushMode(INSIDE);
-CHARACTER       : ~ [:=\r\n\u2028\u2029];
+CHARACTER       : ~ [!#:=\t\f ] -> pushMode(KEY_MODE);
+
+mode KEY_MODE;
+
+KEY_DELIMITER   : [ ]* [:=\t\f ] [ ]* -> type(DELIMITER), popMode, pushMode(VALUE_MODE);
+KEY_SLASH       : '\\' -> more, pushMode(INSIDE);
```

Comment:
It looks like multiline keys are now supported - please add a test case.
I checked it also in Spring application - multivalue keys are supported so it is ok.

```
  @Test
  void shouldParseMultilineKey() {
    var code = """
      multiline\\
         key=This is multiline key value""";

    parseProperties(code);

    assertThat(visitor.visited()).containsExactly(
      "visitPropertiesFile multilinekey=This is multiline key value<EOF><EOF>",
      "visitRow multilinekey=This is multiline key value<EOF>",
      "visitLine multilinekey=This is multiline key value<EOF>",
      "visitKey multilinekey",
      "visitKey This is multiline key value",
      "visitEol <EOF>");
  }
```